tests/riscv64/efficientnet_lite0_quant_rvv_test.run

```diff
@@ -0,0 +1,34 @@
+// RUN: ${TEST_MODULE_CMD} --module_file="${BUILD_RISCV_DIR}/efficientnet_lite0_int8_2.tflite.rvv.vmfb" \
+// RUN:   --entry_function=main --function_input="1x224x224x3xui8=0" | FileCheck %s
+// CHECK: 1x1000xi8=[
```

Comment:
Hey Geoffrey and Scott, Cindy mentioned that we could use x86 as a reference execution and compare RVV against it. However, we would have to compile the models twice. I think that shouldn't be a big deal and will give us better correctness coverage. Would you have any concerns about that?

Reply:
Based on https://github.com/iree-org/iree/runs/8219816244?check_suite_focus=true, it takes 72 seconds to run smoke.sh. Adding the x86_64 vmfb + iree-run-module for these models should add another 1 minute to the test time. Maybe it is acceptable?

Reply:
oh, wait... I thought we would need to recompile IREE (which doesn't make sense). Just compiling the model and running it should be fine, I think!

Reply:
Add x86_64 numerical compare for quant module in a new commit. However, we can't run direct diff for the floating modules due to the numerical error between x86_64 float and riscv emulator float. :(

Reply:
If we use the check framework instead of lit / diff, we can test floating point programs.